lstchain/data/onsite_camera_calibration_param.json

```diff
@@ -1,45 +1,48 @@
 {
-   "version": 1,
-   
-   "CalibrationHDF5Writer": {
-     "minimum_hg_charge_median": 5000,
-     "maximum_lg_charge_std": 300,
-     "one_event": true,
-     "flatfield_product": "FlasherFlatFieldCalculator",
-     "pedestal_product": "PedestalIntegrator",
-     "r0calibrator_product": "LSTR0Corrections",
-     "log_level":"DEBUG"
-   },         
+  "version": 1,
+
+  "CalibrationHDF5Writer": {
+    "minimum_hg_charge_median": 5000,
+    "maximum_lg_charge_std": 300,
+    "one_event": true,
+    "flatfield_product": "FlasherFlatFieldCalculator",
+    "pedestal_product": "PedestalIntegrator",
+    "r0calibrator_product": "LSTR0Corrections",
+    "log_level":"DEBUG"
+  },
+  "EventSource": {
```

Comment:
what was exactly the problem with this configuration file, that it was not containing `max_events`? I do not think that including by default `max_events = 1000` is a good solution here...

Reply:
without `max_event` I get the following error:
```
Traceback (most recent call last):
  File "/usr/local/bin/lstchain_data_create_calibration_file", line 11, in <module>
    load_entry_point('lstchain', 'console_scripts', 'lstchain_data_create_calibration_file')()
  File "/home/yves/ctasoft/cta-lstchain/lstchain/tools/lstchain_data_create_calibration_file.py", line 336, in main
    exe.run()
  File "/home/yves/.local/lib/python3.7/site-packages/ctapipe/core/tool.py", line 199, in run
    self.start()
  File "/home/yves/ctasoft/cta-lstchain/lstchain/tools/lstchain_data_create_calibration_file.py", line 175, in start
    if count == self.tot_events-1 or count == self.eventsource.max_events-1:
TypeError: unsupported operand type(s) for -: 'NoneType' and 'int'
```
An other option would to change the code in lstchain_data_create_calibration_file.py to use all event if `max_event` is not present

Reply:
Add a `self.eventsource.max_events is not None` as first check.

Reply:
```
max_events_reached = self.eventsource.max_events is not None and count == self.eventsource.max_events - 1
if count == self.tot_events - 1 or max_events_reached
```